mailbox/jpa/src/main/java/org/apache/james/mailbox/jpa/JPAId.java

```diff
@@ -25,7 +25,7 @@
     public static class Factory implements MailboxId.Factory {
         @Override
         public JPAId fromString(String serialized) {
-            return of(Long.parseLong(serialized));
+            return of(Long.valueOf(serialized));
```

Comment:
I think you should be a little bit more careful about your historing while stashing and fixing. I think it could be fine for this one.